src/api/l_audio.c

```diff
@@ -150,15 +157,43 @@ static const luaL_Reg lovrAudio[] = {
   { "setCaptureDevice", l_lovrAudioSetCaptureDevice },
   { "getCaptureDevice", l_lovrAudioGetCaptureDevice },
   { "listCaptureDevices", l_lovrAudioListCaptureDevices },
+  { "getSpatializerName", l_lovrAudioGetSpatializerName },
   { NULL, NULL }
 };
 
 int luaopen_lovr_audio(lua_State* L) {
   lua_newtable(L);
   luax_register(L, lovrAudio);
   luax_registertype(L, Source);
-  AudioConfig config[2] = { { .enable = true, .start = true }, { .enable = false, .start = false } };
-  if (lovrAudioInit(config)) {
+
+  char *spatializer = NULL;
+  int spatializerMaxSourcesHint = AUDIO_SPATIALIZER_MAX_SOURCES_HINT;
+  luax_pushconf(L);
+  lua_getfield(L, -1, "audio");
+  if (lua_istable(L, -1)) {
+    lua_getfield(L, -1, "spatializerMaxSourcesHint");
+    if (lua_type(L, -1) == LUA_TNUMBER)
+      spatializerMaxSourcesHint = lua_tointeger(L, -1);
+    lua_pop(L, 1);
+
+    lua_getfield(L, -1, "spatializer");
```

Comment:
- Is it weird you set this with "spatializer" but read it back with "spatializerName"?
- Should this optionally accept a table list of names rather than a single string name?

Reply:
- I think it's okay
- table vs. string thoughts
  - If I set the spatializer to 'oculus' but accidentally run it on a build without the spatializer compiled in, should I get an error or fall back to the null spatializer?
  - Do I ever want to build in multiple spatializers and have it try multiple (maybe for a cross-platform project)?  This seems unlikely because I am the one compiling in the spatializers and it seems rare for a spatializer to be "unsupported" once it is compiled in and tested.  Dynamically changing the spatializer seems like a choice I can make at packaging time, not run time.
  - I guess I lean towards string, or even just a boolean (aka no selection, spatializer depends on build).